ui/src/telemetry/process.worker.ts

```diff
@@ -0,0 +1,35 @@
+import { API, Table } from "./constants"
+
+import type { WorkerPayloadShape } from "types"
+import { fetchApi } from "utils"
+import * as QuestDB from "utils/questdb"
+
+const start = async (payload: WorkerPayloadShape) => {
+  const quest = new QuestDB.Client(payload.host)
+
+  const result = await quest.queryRaw(`
+    SELECT *
+      FROM ${Table.MAIN}
+      WHERE created > to_date('${payload.lastUpdated}', 'yyyy-MM-ddTHH:mm:ss.SSSZ')
```

Comment:
We should change `telemetry` table structure to have `created` as timestamp. This will enable this SQL to become:

```sql
telemetry where created > '2020-06-29T21:46:32.973Z'
```

with designated timestamp the above SQL will become couple of orders of magnitude more efficient

Reply:
Sounds good :)